BuildTools/CMake/README.md

```diff
@@ -0,0 +1,20 @@
+# rocBLAS-Examples CMake
+This rocBLAS example shows the use of CMake to build an excutable that utilizes rocBLAS.  The main.cpp C++ code illustrates nothing new, and the standard Makefiles provide for the entire rocBLAS-Examples make build system so they can be ignored.  Thus the only files of interest are the CMake files CMakeLists.txt and src/CMakeLists.txt.   The CMake project depends only on the rocBLAS package which automatically brings in the rocBLAS dependencies for hip to allow inclusion of hip header files in main.cpp without explicit include search paths being added.  This example assumes you are already familiar with CMake as it will not explain the structure.
```

Comment:
```suggestion
This example shows how to use rocBLAS in a C++ program with a CMake build system.  The focus for this example is the setup in [CMakeLists.txt](CMakeLists.txt) and [src/CMakeLists.txt](src/CMakeLists.txt).  The C++ code in `main.cpp` is just a trivial example of a rocBLAS call, and the Makefiles are just the part of the rocBLAS-Examples make build system, so they can be ignored.  The CMake project depends only on the rocBLAS package, which automatically brings in the rocBLAS dependencies for hip. This allows inclusion of hip header files in `main.cpp` without having to explicitly specify include search paths. 
```

I edited it a bit for grammar, but mostly to focus on what the example is about.